src/renderers/TextRenderer.js

```diff
@@ -31,10 +31,16 @@ class TextRenderer extends SketchRenderer {
       props.resizingConstraint
     );
 
-    const resolvedStyle = TextStyles.resolve(textStyle);
-    if (resolvedStyle) {
-      layer.style = resolvedStyle.sketchStyle;
-      layer.style.sharedObjectID = resolvedStyle.sharedObjectID;
+    const resolvedTextStyle = TextStyles.resolve(textStyle);
+    if (resolvedTextStyle) {
+      layer.style = resolvedTextStyle.sketchStyle;
+      layer.style.sharedObjectID = resolvedTextStyle.sharedObjectID;
+    } else {
```

Comment:
you are probably right, I just didn't want to change the behavior if it's working in some unforeseen cases. A shared style might specify the `flexDirection` perhaps and it would match here?

`inheritableStyles.flexDirection = 'row'; ` was introduced when we switched to `yoga-layout`. I didn't really look into why tbh.

Do you mind merging this PR which does the job (and that we need to release very soon) and I'll create an issue for cleaning that part later?

Reply:
> A shared style might specify the flexDirection perhaps and it would match here?

I thought the same thing, but then looked at where the registered style was added and it ensures only "safeStyles" are hashed: https://github.com/airbnb/react-sketchapp/blob/1974091c696470138216a54611abd125d5828af9/src/sharedStyles/TextStyles.js#L28

Reply:
> Do you mind merging this PR which does the job (and that we need to release very soon) and I'll create an issue for cleaning that part later?

Sure. Don't see any harm in doing so.

Reply:
Ah indeed! So maybe the right fix is the `pick(style, INHERITABLE_FONT_STYLES)` when resolving as well

Reply:
That would indeed be a quick and accurate fix.

But if this is the fix.... why should `textStyle` be created on the node at all?

Reply:
I mean just look up the shared style when resolving. We still want to pass all the text styles I thing

Reply:
I guess my point was that textStyle is more of a computed prop than something that actually needs to be stored on the node.

Will have to do more digging, but I think your solution is a good first pass. And then we could always create a new issue to investigate the need for `textStyle` prop.